### PR TITLE
open source contribute

### PR DESCRIPTION
**Pull request by Gunjan-Rajput**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% assign t = site.data.locales[page.lang][page.lang] %}
-<button id="scrollToTopButton" title="Go to top">Scroll to Top</button>
+
+<button id="scrollToTop" title="Go to top">Scroll to Top</button>
```
Comment (olsza) [inline _includes/footer.html:3]: Why did you change the ID attribute from "scrollToTopButton" to "scrollToTop"? Because of this change it does not work as it should.

File: _includes/footer.html [hunk 2]
```diff
@@ -36,8 +37,8 @@ <h3 class="h3-mktg mb-3">{{ t.footer.subscribe.heading }}</h3>
                     class="form-input required email d-block col-10 mx-auto py-2 px-3 mb-3" id="mce-EMAIL"
                     autocomplete="home email">
                   <input type="checkbox" value="1" name="group[9617][1]" id="mce-group[9617]-9617-0" checked="checked"
-                    style="display:none">
-                  <input type="submit" value="{{ t.footer.subscribe.button }}" name="subscribe"
+                    style="display:none
```
Comment (olsza) [inline _includes/footer.html:40]: Or you change the PR to draft status if you're still working on it, but also check locally because it's a bug.
```suggestion
                    style="display:none">
```

**Comment by Gunjan-Rajput** [comment]
I change. Please  check.
